### PR TITLE
Add documents for testbed disconnect and destroy features usage

### PR DESCRIPTION
**Pull request by akhosrav**

### CODE REVIEW

**Review by ThomasJRyan — APPROVED** [review]

File: docs/topology/usage.rst [hunk 1]
```diff
@@ -142,6 +142,88 @@ This is a convenience function, as under the hood it uses threads perform
 per device ``.connect()`` asynchronously. All other behavior follows that of
 the above connection manager concept. 
 
+Disconnect From All Devices
+----------------------
+
+``Testbed`` object also provides a convenience function, ``Testbed.disconnect()``,
```
Comment (jeaubin) [inline docs/topology/usage.rst:148]: remove the also

File: docs/topology/usage.rst [hunk 2]
```diff
@@ -142,6 +142,88 @@ This is a convenience function, as under the hood it uses threads perform
...
+allowing you to make asynchronous disconnection from multiple testbed devices
+at the same time.
+
+.. code-block:: python
+    # Example
```
Comment (jeaubin) [inline docs/topology/usage.rst:154]: i believe you need a space between code block and the actual code

File: docs/topology/usage.rst [hunk 3]
```diff
@@ -142,6 +142,88 @@ This is a convenience function, as under the hood it uses threads perform
...
+    # -------
+    #
+    #   disconnecting from devices in parallel
```
Comment (jeaubin) [inline docs/topology/usage.rst:157]: extra space before disconnecting

File: docs/topology/usage.rst [hunk 4]
```diff
@@ -142,6 +142,88 @@ This is a convenience function, as under the hood it uses threads perform
...
+    # using the sample topology file from
+    from pyats import topology
+    testbedfile = os.path.join('sampleTestbed.yaml')
```
Comment (jeaubin) [inline docs/topology/usage.rst:160]: keep it simple, just say

    testbed = topology.loader.load('your-testbed-file.yaml')

you arent joining a path anyway

File: docs/topology/usage.rst [hunk 5]
```diff
@@ -142,6 +142,88 @@ This is a convenience function, as under the hood it uses threads perform
...
+    # disconnect from some devices in this testbed
+    # using unique vias per device, and shared kwargs
+    # shared keyword-arguments will be passed to every single connection
+    testbed.disconnect(testbed.devices['uut'],
+                    testbed.devices['helper'],
+                    vias = {'uut': 'cli',
+                            'helper': 'console'},
+                            log_stdout = False)
+
+This is a convenience function, as under the hood it uses threads perform
```
Comment (jeaubin) [inline docs/topology/usage.rst:184]: This is a convenience function using threads under the hood to perform ...

**Review by LukasMcClelland — APPROVED** [review]

File: docs/topology/usage.rst [hunk 6]
```diff
@@ -142,6 +142,88 @@ This is a convenience function, as under the hood it uses threads perform
...
+per device ``.disconnect()`` asynchronously.
+
+Destroy connections to all device
+----------------------
+
+``Testbed`` object also provides a convenience function, ``Testbed.destroy()``,
+allowing you to destroy connections asynchronously from multiple testbed
+devices.
```
Comment (jeaubin) [inline docs/topology/usage.rst:192]: can you explain the difference between disconnect/destroy - high level

**Review by jeaubin — APPROVED** [review]

**Pull request merged**